assets/extensions/examples/online-extensions.js

```diff
@@ -0,0 +1,92 @@
+/*
+  Override the CLI download locations
+*/
+window.OPENSHIFT_CONSTANTS.CLI = {
+  "Linux (32 bits)": "https://openshift.com/tbd_url/32bitlinux.tar.gz",
+  "Linux (64 bits)": "https://openshift.com/tbd_url/64bitlinux.tar.gz",
+  "Windows":         "https://openshift.com/tbd_url/win.zip",
+  "Mac OS X":        "https://openshift.com/tbd_url/mac.zip"
+};
+
+/*
+  Override the documentation links
+*/
+window.OPENSHIFT_CONSTANTS.HELP = {
+  "cli":                     "https://docs.openshift.com/online/latest/cli_reference/overview.html",
+  "get_started_cli":         "https://docs.openshift.com/online/latest/cli_reference/get_started_cli.html",
+  "basic_cli_operations":    "https://docs.openshift.com/online/latest/cli_reference/basic_cli_operations.html",
+  "webhooks":                "https://docs.openshift.com/online/latest/dev_guide/builds.html#webhook-triggers",
+  "new_app":                 "https://docs.openshift.com/online/latest/dev_guide/new_app.html",
+  "start-build":             "https://docs.openshift.com/online/latest/dev_guide/builds.html#starting-a-build",
+  "deployment-operations":   "https://docs.openshift.com/online/latest/cli_reference/basic_cli_operations.html#build-and-deployment-cli-operations",
+  "route-types":             "https://docs.openshift.com/online/latest/architecture/core_concepts/routes.html#route-types",
+  "persistent_volumes":      "https://docs.openshift.com/online/latest/dev_guide/persistent_volumes.html",
+  "compute_resources":       "https://docs.openshift.com/online/latest/dev_guide/compute_resources.html",
+  "default":                 "https://docs.openshift.com/online/latest/welcome/index.html"
+};
+
+/*
+  Custom angular module 
+*/
+angular.module('openshiftOnlineConsoleExtensions', ['openshiftConsole'])
+  .run(function(HawtioExtension) {
+
+    /*
+      Request system status from statuspage.io
+    */
+    var system_status_elem = $('<a href="http://status.openshift.com/" target="_blank" class="nav-item-iconic system-status project-action-btn" style="display: none;">');
```

Comment:
Nope if you are creating a single node jquery doesn't require an end tag,
so you can just do `$('<a>')` and whatever attributes
On Feb 10, 2016 11:57 AM, "Sam Padgett" <notifications@github.com> wrote:

> In assets/extensions/examples/online-extensions.js
> <https://github.com/openshift/origin/pull/7169#discussion_r52487491>:
>
> > +  "route-types":             "https://docs.openshift.com/online/latest/architecture/core_concepts/routes.html#route-types",
> > +  "persistent_volumes":      "https://docs.openshift.com/online/latest/dev_guide/persistent_volumes.html",
> > +  "compute_resources":       "https://docs.openshift.com/online/latest/dev_guide/compute_resources.html",
> > +  "default":                 "https://docs.openshift.com/online/latest/welcome/index.html"
> > +};
> > +
> > +/*
> > +  Custom angular module
> > +*/
> > +angular.module('openshiftOnlineConsoleExtensions', ['openshiftConsole'])
> > +  .run(function(HawtioExtension) {
> > +
> > +    /*
> > +      Request system status from statuspage.io
> > +    */
> > +    var system_status_elem = $('<a href="http://status.openshift.com/" target="_blank" class="nav-item-iconic system-status project-action-btn" style="display: none;">');
>
> Should this have the end tag </a>?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/7169/files#r52487491>.
>